src/Winium.Cruciatus/Application.cs

```diff
@@ -109,9 +120,32 @@ public void Start(string arguments)
                                Arguments = arguments
                            };
 
-            this.process = Process.Start(info);
+            this.process = Process.Start(info);           
         }
 
+        /// <summary>
+        /// Update process property by process name
+        /// </summary>
+        /// <param name="processName">Launched application process name</param>
+        /// <returns>true if launched process is successfully update. false if there is error occurs</returns>
+        public void UpdateProcessBy(string processName)
```

Comment:
To match the function description `Update process property by process name` then let's name the function as its meaning `UpdateProcessBy` --> `UpdateProcessByName`